Simplenote/Classes/SPTagHeaderView.swift

```diff
@@ -19,6 +19,16 @@ class SPTagHeaderView: UIView {
         }
     }
 
+    /// Message label
+    @IBOutlet private(set) var messageLabel: UILabel! {
+        didSet {
+            messageLabel.text = NSLocalizedString(
+                "Once you add tags to your notes, your tags will appear here.",
+                comment: "Message displayed when no tags exist"
+            )
+        }
+    }
+
```

Comment:
Potentially speaking, a specific UITableViewCell specifically implemented for this is a best fit.

**Why:** UITableView is known to have issues whenever the TableSectionHeaderView changes its height, by using a cell we'd just avoid the entire issue.

Reply:
As per the design there is no separator between the section header and the message cell. However currently the table view has its `separatorStyle` set to `singleLine`. To mix and match separator style would require manually adding the separators to the cells which might complicate things for us. Do we want to go through that much to add just a message here?